### ISSUE TITLE
Script for retrain inception model

### ISSUE THREAD

**Issue by skrealworld** [RITficialIntelligence/kaggle-cancer#5]

**Comment by skrealworld** [comment]
Sample code 
https://www.kernix.com/blog/image-classification-with-a-pre-trained-deep-neural-network_p11

**Comment by skrealworld** [comment]
First training accuracy : 96 % 
First validation accuracy : 56 % (On the data which is 10 % of training data)
